packages/storefront-api-client/src/storefront-api-client.ts

```diff
@@ -100,12 +101,12 @@ export function createStorefrontApiClient({
     getApiUrl,
   });
 
-  const fetch = (...props: ApiClientRequestParams) => {
+  const fetch: ApiClientFetch = (...props) => {
```

Comment:
We can do the same thing here as the `AdminApiClient` 😄  - one less change for the future PR 😉

Reply:
Already done in a separate branch for SFAPI types :)